assignments/assignment0.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Assignment 0 - GIT Instructions
+nav_exclude: true
+---
+
+# GIT Instructions
+
+## 1. Set Up Your GitHub Repository:
+- Create a new repository on GitHub named `username.github.io`, where `username` is your GitHub username.
+- This repository will host your personal website.
+
+## 2. Design Your Website:
+- Create a simple HTML/CSS website. You can use any website template you like or build your own from scratch.
+- Ensure that your website includes the following sections:
+  - **About Me**: A brief introduction about yourself.
+  - **Work Experience**: If any, highlight your work experience.
+  - **Projects**: A section showcasing some of the projects you've worked on.
+  - **Contact**: Information on how to reach you, like email, LinkedIn, etc.
+  - Each section should be a separate page. Don't include multiple sections in same page, create different dedicated pages for different sections.
+
+## 3. Create an Assignment Section:
+- Create a page for an **Assignment** section where you will put all your course assignments.
+- For each assignment, you will provide a brief explanation of how you approached it.
+- You will include a link to the GitHub code repository of that assignment.
+- Below is an example image of how your assignment page in your portfolio website might look like (Styling is not necessary, you can choose to keep minimal styling):
+![assignments_image](../../assets/images/hw1-image.png)
+
+## 4. Conventions and Rules:
+Don't forget to follow the given rules:
+- The href/link to your assignment's github code repo should be named : "Github Link". This is a strict naming convention, and should be followed.
```

Comment:
```suggestion
Graders will be running an automated script to scrape all websites for each assignment. As such it's critical that:
1. your assignment repo be public after the deadline (not before)
2. the href/link to your assignment's github repo on your website is exactly : "Github Link". Note: capitalization is important here.
3. The name of your github code repository should be of the format: "assignment-X" (where X is the assignment number), for example: "assignment-1".

If any of these is missing, graders will not be able to find your assignment and you will get 0. For the first few assignments, if you do get 0 because the grading staff was unable to find your assignment, you will have 24h after grades are returned to fix the issue for a regrade. This second chance will not be available after the first month of classes.
```